madmin/templates/map.html

```diff
@@ -283,6 +283,29 @@
 				$this.addLayer("stops with quests", l_quests, false, "quets");
 			});
 		},
+		map_fetch_geofences() {
```

Comment:
This needs to be called in the `map_fetch_everything()` method to actually fetch the geofences.

Reply:
Jep. Somehow slipped through my manual merge